crates/spfs/src/config.rs

```diff
@@ -587,11 +587,11 @@ impl Config {
         match self.remote.get(remote_name.as_ref()) {
             Some(Remote::Address(remote)) => {
                 let config = RemoteConfig::from_address(remote.address.clone()).await?;
-                tracing::debug!(?config, "opening repository");
+                tracing::debug!(?config, "opening '{}' repository", remote_name.as_ref());
```

Comment:
```suggestion
                tracing::debug!(?config, "opening '{}' repository via address", remote_name.as_ref());
```

This change looks fine to me, but there's also an opportunity here to disambiguate these two trace lines so we can tell which path was taken from the log. The other one could say "via config" for symmetry.

Reply:
I've added different text to each log message.